packmanager/umbrella.go

```diff
@@ -295,3 +297,30 @@ func NewUmbrellaManager(ctx context.Context, constructors []func(*UmbrellaManage
 
 	return u, nil
 }
+
+func (u UmbrellaManager) Show(ctx context.Context, outputFormat string, qopts ...QueryOption) (any, error) {
+	var metadatas []any
+	var errList []string
+
+	for _, manager := range u.packageManagers {
+		log.G(ctx).WithFields(logrus.Fields{
+			"format": manager.Format(),
+		}).Tracef("showing package information")
+
+		metadata, err := manager.Show(ctx, outputFormat, qopts...)
+		if err != nil {
+			errList = append(errList, err.Error())
+			log.G(ctx).
+				WithField("format", manager.Format()).
+				Debugf("could not run Show(): %v", err)
+			continue
+		}
+
+		metadatas = append(metadatas, metadata)
+	}
+	if len(metadatas) == 0 {
+		return nil, fmt.Errorf(strings.Join(errList, "\n"))
+	}
```

Comment:
I would not propagate both errors every time.
If the package is not found, just specify that no information was found about the package:

```suggestion
	if len(metadatas) == 0 {
		return nil, fmt.Error("no information found about requested package")
	}
```

![image](https://github.com/unikraft/kraftkit/assets/46280801/47380f8c-8cf2-4914-8cb3-0c1ecb80857d)

Reply:
Done! 👍🏽